indexers/staking-squid/src/events/operator.ts

```diff
@@ -64,6 +64,7 @@ export function processOperatorNominatedEvent(
       nominatorId: nominator.id,
       amount,
       storageFeeDeposit,
+      totalAmount: amount + storageFeeDeposit,
```

Comment:
I don't understand why totalAmount is calculated this way: 

Shouldn't be `totalAmount` be the sent amount in the extrinsic? Or what's the same, the actual deposit (`amount` in the code) be extrinsic `amount` arg minus `storageFeeDeposit`?

Reply:
[This is the code](https://github.com/autonomys/subspace/blob/main/crates/pallet-domains/src/bundle_storage_fund.rs#L142 ) that regulates this mechanism. I think that it separates the extrinsic `amount` in two parts.

Reply:
Very good point, indeed my logic was wrong!
Thanks